docs/filters.md

```diff
@@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
 
 *Expects a string*
 
-Matches all issues associated with any milestones whose names contain the given string.
+Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
 
```

Comment:
I don't allow, to make it consistent with `prev-` since I think the current
regex will read ` - ` as negation.
On Oct 12, 2015 10:29 PM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

> In docs/filters.md
> <https://github.com/HubTurbo/HubTurbo/pull/1050#discussion_r41761778>:
>
> > @@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
> >
> >  *Expects a string*
> >
> > -Matches all issues associated with any milestones whose names contain the given string.
> > +Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
> >
>
> Perhaps variable is the wrong term. I meant it is a keyword and we have no
> way to show the difference. Still, may be simpler to keep as it is.
> Certainly easier to type.
> @HansNewbie <https://github.com/HansNewbie> do we allow spaces around +?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1050/files#r41761778>.
>